packages/esm-appointments-app/src/home/index.tsx

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
-
-import AppointmentsBaseTable from './appointments-base-table.component';
+import { useTranslation } from 'react-i18next';
+import AppointmentsList from '../appointments/scheduled/appointments-list.component';
+import dayjs from 'dayjs';
+import styles from './index.scss';
+import Overlay from '../overlay.component';
 
 const HomeAppointments = () => {
-  return <AppointmentsBaseTable />;
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.container}>
+      <AppointmentsList
+        date={dayjs().startOf('day').format('YYYY-MM-DDTHH:mm:ss.SSSZZ')}
```

Comment:
Hmmm... We should probably make the `isoFormat` constant importable... and change [`DateInput`](https://github.com/openmrs/openmrs-esm-core/blob/3ce509b6b4b78e070c829793f91c5e995ad88d9c/packages/framework/esm-utils/src/omrs-dates.ts#L19) to indicate that it can take a `dayjs.Dayjs` object (which would allow just calling `toOmrsIsoString(dayjs().startOf('day'))`.

Reply:
makes sense, for now I just changed to:
toOmrsIsoString(dayjs().startOf('day').toDate()).